Feature/10 help in html

Justyna-JustCode opened the pull request:
The help document can be reviewed from this link: http://just-code.org/DATA/applications/resto/help.html

Mariusz112358 commented:
In the first paragraph, I have badly rendered characters:

> â€žRestoâ€ is a Spanish first person singular form of a verb â€žrestarâ€, which can be translated as â€žto remainâ€. There is also an obvious phonetic connection to an English verb â€žto restâ€.

In the second paragraph, I would change the first part to (before "Additionally"):

> Sitting in a front of a computer for several hours may be a cause of various health issues (see for example https://www.betterhealth.vic.gov.au/health/healthyliving/the-dangers-of-sitting). Physical inactivity is considered one of the top causes of premature death around the world (see https://en.wikipedia.org/wiki/Preventable_causes_of_death). To keep you in a good shape and let you remain focused throughout the day it is important to take regular breaks from sitting. Just stand up for a while, take a short walk and try not to think about your work issues. The Resto will inform you about a break as it is hard to be focused and at the same time track passing time.

Justyna-JustCode commented:
Thanks for your input. I like your proposal for the 2nd paragraph (I've updated it a little bit).
The wrong displayed in the 1st paragraph is also fixed as well as the stop button description in the 3rd paragraph.
I skipped the change in the image for "Main window (idle)". I used plural form because there are multiple buttons like this but adding more arrows would make an image harder to understand - there are already multiple arrows there.

Please take a look on the newest version.
